com.gitbutler.app.yml

```diff
@@ -0,0 +1,79 @@
+id: com.gitbutler.app
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: git-butler
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=session-bus
```

Comment:
`session-bus` isn't allowed. You need to explicitly specify the DBus names needed.